ultravox/tools/ds_tool.py

```diff
@@ -0,0 +1,172 @@
+import dataclasses
+import os
+from typing import Any, Dict, Optional, Union
+
+import datasets
+import openai
+import simple_parsing
+
+from ultravox.tools import tts
+
+CHAT_CLIENT = openai.Client()
+TTS_CLIENT = tts.AzureTts()
+
+DEFAULT_TEXTGEN_TEMPLATE = """Passage: {passage}
+
+Question: {question}
+
+Answer: {answer}
+
+Provide a short explanation to the question given the passage that entails the answer."""
```

Comment:
"entails" is an unusual choice here. Maybe "Given the question and passage, provide a short explanation for the provided answer" or some such. 

Also, this is fine for now, but it seems hard to have a useful default for this sort of thing.

Reply:
> "entails" is an unusual choice here.

You might be right.
Just to explain though, the idea is to be able to use it so that Ultravox outputs: `{explanation} \n Answer: {answer}`.  In this way, we want the generated "explanation" be able to "precede" the answer. That's why I wrote the prompt like so. Text generated via the proposed prompt might cause the wording to be slightly incorrect.

> it seems hard to have a useful default for this sort of thing.

I agree. I didn't know where to put the info though. For Expresso, I put it in the comments. I could alternatively put this info in the dataset README as well, but that might not be enough.

Reply:
btw I'm not attached to the prompt though. If you think the new version is better given the explanation above then I'll change it.

Reply:
I'm still a bit confused regarding how the explanation is to be used. But mainly I was reacting to "entails" and so maybe a smaller change would be "entails" -> "provides a rationale for"

Reply:
Done.